CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 * **[Breaking change]** Remove `AppCenter.setCustomProperties` API.
 
+### App Center Analytics
+
+#### Android/iOS
+
+ * **[Feature]** Add `Analytics.enableManualSessionTracker` and `Analytics.startSession` APIs for control start session time manually.
```

Comment:
Please use android's text for the consistency: **[Feature]** Add `Analytics.enableManualSessionTracker` and `Analytics.startSession` APIs for tracking session manually.